site/content/integrate/apps/structure/manifest/_index.md

```diff
@@ -1,54 +1,61 @@
-| `description`           | string      | The description for your app. Used in the Marketplace. Provide examples of key functionality the App provides in a short paragraph.      |
-| `icon`                  | string      | The icon for your app. Used as the bot account icon and in the Marketplace. A relative path in the static assets folder of a .PNG image. |
-| `requested_permissions` | Permissions | All the permissions needed by the app.                                                                                                   |
-| `requested_locations`   | Locations   | The list of top-level locations that the application intends to bind to.                                                                 |
-| `bindings`              | Call        | The call gets invoked to retrieve bindings. By default invoke `/bindings`.                                                               |
-| `on_disable`            | Call        | The call gets invoked when the app is disabled after having been enabled.                                                                |
-| `on_enable`             | Call        | The call gets invoked when the app is enabled after having been disabled.                                                                |
-| `on_install`            | Call        | The call gets invoked when the app gets installed.                                                                                       |
-| `on_uninstall`          | Call        | The call gets invoked when the app gets uninstalled, before the app is actually removed.                                                 |
-
-- (*) `app_id` and `homepage_url` must be provided and valid.
-- `on_...` callbacks are not invoked unless explicitly provided in the manifest.
-`bindings` defaults to `/bindings` with nothing expanded.
-- In addition, an app must define at least one of the following deployment types, as a section
-in its `manifest.json`: `"http"`, `"aws_lambda"`, `"kubeless"`, or
-`"open_faas"`.
-
-An example manifest looks like this:
+All Apps should define a manifest ({{<newtabref title="godoc" href="https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#Manifest">}}) as a JSON object.
+The fields of the manifest are described in the following table:
+
+| Name                                       | Type                        | Description                                                                                                                                             |
+|:-------------------------------------------|:----------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `app_id`{{<compass-icon icon-star>}}       | string                      | ID for your App.                                                                                                                                        |
```

Comment:
@neflyte I see you've dropped the asterisk (*) in favor of a star icon. My main concern with this approach is accessibility for screenreaders, because the icon has no alt text. The asterisk is usually read aloud by the screenreader, but in this case it would be skipped, or the reader wouldn't know what it is.

Generally, I think using an asterisk is much more universally recognizable, but I'm 1/5 on it.

cc @cwarnermm

Reply:
I added tag attributes to support alt text and screen readers. Were they not present?

Reply:
I see what happened. I put in the alt tags and screen reader support for the same icons....in the product docs site. ;)
I'll add them here as well.

Great catch! :smile: